src/transformStateWithClones.js

```diff
@@ -8,6 +8,36 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const stateCopy = { ...state };
+  const result = [];
+
+  for (const action of actions) {
+    switch (action.type) {
```

Comment:
Use destructuring for `action`
```suggestion
  for (const action of actions) {
    const { type, ..... } = action;
    
    switch (type) {
```